gnu-apl-editor.el

```diff
@@ -159,7 +169,8 @@ successfully."
         (gnu-apl--send-new-function parts tag)))))
 
 (defun gnu-apl--remove-final-endfn (strings)
-  "If the last element is ∇, return a new list with that element removed, else return the original list."
+  "Remove ∇ if it is the last element in STRINGS.
```

Comment:
While I generally don't have a problem with rewrites of docstrings, this change makes it seem as though the string is modified in-place, which is not the case.

Reply:
How about "Remove ∇ non-destructively ..."?

Reply:
I suggest: `Return STRINGS without the terminating ∇, if present`

Reply:
Done.